src/components/NewWorkspaceModal.js

```diff
@@ -371,6 +373,34 @@ const NewWorkspaceModal = withDisplayName(
                     }),
                   ]),
               ]),
+              isGoogleBillingProject() &&
+                h(IdContainer, [
+                  (id) =>
+                    h(Fragment, [
+                      h(FormLabel, { htmlFor: id }, [
+                        h(Checkbox, {
+                          'aria-label': 'enhancedBucketLoggingToggle',
+                          style: { margin: '0.25rem 0.25rem 0.25rem 0' },
+                          id,
+                          checked: enhancedBucketLogging,
+                          disabled: requireEnhancedBucketLogging || groups.length > 0,
+                          onChange: () => setEnhancedBucketLogging(!enhancedBucketLogging),
+                        }),
+                        'Enhanced Bucket Logging',
+                        h(InfoBox, { style: { marginLeft: '0.25rem', padding: '0' } }, [
+                          '"Enhanced Bucket Logging" means ...',
+                          h(
+                            Link,
+                            {
+                              href: 'https://youtu.be/E4WlUXrJgy4',
```

Comment:
Can we get a real link in here?

Reply:
Absolutely before it gets merged! I don't think we have a support article for Enhanced Bucket Logging yet. How would we get that done?

Reply:
I guess ask your product person/user support?  But even without a full article maybe we can put 2-3 sentences here to give a rough idea?

Reply:
Wrote a blurb.